### PR TITLE
Default family

### PR DESCRIPTION
**Pull request by alexeymorozov**
See fontist/fontist#247

### CODE REVIEW

File: lib/metanorma/ogc/processor.rb [hunk 1]
```diff
@@ -25,8 +25,8 @@ def fonts_manifest
           "Arial" => nil,
           "STIX Two Math" => nil,
           "Source Han Sans" => nil,
+          "Source Han Sans Normal" => nil,
           "Fira Code" => nil,
```
Comment (alexeymorozov) [inline lib/metanorma/ogc/processor.rb:29]: @Intelligent2013 Do we need "Fira Code" subfamilies? "Fira Code Light"

File: lib/metanorma/ogc/processor.rb [hunk 2]
```diff
@@ -25,8 +25,8 @@ def fonts_manifest
           "Arial" => nil,
```
Comment (alexeymorozov) [inline lib/metanorma/ogc/processor.rb:25]: @Intelligent2013 Lato also has other subfamilies, like Lato Light. Any chance we need them?

File: lib/metanorma/ogc/processor.rb (hunk 1 above)
Comment (Intelligent2013) [reply lib/metanorma/ogc/processor.rb:29]: No, we use 'style="normal" weight="normal"' only.

File: lib/metanorma/ogc/processor.rb (hunk 2 above)
Comment (Intelligent2013) [reply lib/metanorma/ogc/processor.rb:25]: Let's add Lato Light.

File: lib/metanorma/ogc/processor.rb (hunk 1 above)
Comment (alexeymorozov) [reply lib/metanorma/ogc/processor.rb:30]: Ok, thank you :)

File: lib/metanorma/ogc/processor.rb (hunk 2 above)
Comment (alexeymorozov) [reply lib/metanorma/ogc/processor.rb:26]: Thanks and done :)

**Pull request merged**